net-libs/libpsl/libpsl-0.19.1.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="C library for the Public Suffix List"
+DESCRIPTION="Libpsl can be used by browsers and other web clients to avoid privay-leaking 'supercookies' or 'super domain' certificates, highlight parts of the domain in a user interface, and sort domain lists by site.'"
+HOMEPAGE="https://rockdaboot.github.io/libpsl/"
+COPYRIGHT="2014-2015 Tim Rühsen"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/rockdaboot/libpsl/releases/download/libpsl-$portVersion/libpsl-$portVersion.tar.gz"
+CHECKSUM_SHA256="735146b51bbd3dcb6b0f87819c64bf3115f7fb9fa2e3a7fe9966e3346a8abc79"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
```

Comment:
This is our convention towards library versioning. This would save future maintainers from editing anything other than `CHECKSUM_SHA256` when your port gets an update

```diff
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+